src/overlays/actors/ovl_Obj_Lupygamelift/z_obj_lupygamelift.c

```diff
@@ -28,35 +33,142 @@ const ActorInit Obj_Lupygamelift_InitVars = {
     (ActorFunc)ObjLupygamelift_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80AF0740[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 200, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 400, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
-
-extern InitChainEntry D_80AF0740[];
-
-extern UNK_TYPE D_060048D0;
-extern UNK_TYPE D_060071B8;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/ObjLupygamelift_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/ObjLupygamelift_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/func_80AF0394.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/func_80AF04BC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/func_80AF04D8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/func_80AF0514.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/func_80AF0530.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/ObjLupygamelift_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Lupygamelift/ObjLupygamelift_Draw.s")
+void ObjLupygamelift_Init(Actor* thisx, GlobalContext* globalCtx) {
+    s32 pad;
+    ObjLupygamelift* this = THIS;
+    Path* path;
+    s32 phi_v0;
+
+    Actor_ProcessInitChain(thisx, sInitChain);
+    this->dyna.actor.scale.y = 0.15f;
+    this->dyna.actor.shape.rot.x = 0;
+    this->dyna.actor.world.rot.x = 0;
+    this->dyna.actor.shape.rot.z = 0;
+    this->dyna.actor.world.rot.z = 0;
```

Comment:
These look like chained assignments, can they be?

Reply:
They are not